portfolio/src/main/webapp/script.js

```diff
@@ -80,8 +80,38 @@ function getComments() {
       commentContainer.className += 'comment-container';
       commentContainer.innerText = message;
 
-      document.getElementById('message-container').appendChild(commentContainer);
+      document.getElementById('message-container')
+          .appendChild(commentContainer);
     }
   });
 }
-getComments();
\ No newline at end of file
+getComments();
+
+/**
+ * Checks login status of user, links to login/logout page, hides comment form
+ * if user is logged out.
+ */
+function checkLoginStatus() {
+  fetch('/login')
+  .then(response => response.json())
+  .then((status) => {
+    const baseUrl = window.location.origin;
+    let url = new URL(status[1], baseUrl);
+
+    let commentForm = document.getElementById('comment-form');
+    let para = document.createElement("p");
+    
+    if (status[0] === "True") {
+      para.innerHTML = "<p>Logout <a href=\"" + url + "\">here</a>.</p>";
+      commentForm.appendChild(para);
+    }
+    else {
+      commentForm.getElementsByTagName('form')[0].style.display = "none";
```

Comment:
Using `style.display` to show/hide things is good when you intend to show/hide dynamically.  In this case, it decided whether to show/hide this feature when you first load the script. For this case, I believe this function should decide whether to have the form element in DOM or not. This means you would remove the form from the HTML and need a method to create the form element and everything inside of it in JS.... seems pretty annoying ngl. 

I would like Andrew's take on this.